src/github/pagination.ts

```diff
@@ -20,8 +53,8 @@ async function paginate (octokit: GitHubAPI, responsePromise: any, callback = de
   collection = collection.concat(callback(response, done))
 
   // eslint-disable-next-line no-unmodified-loop-condition
-  while (getNextPage && octokit.hasNextPage(response)) {
-    response = await octokit.getNextPage(response)
+  while (getNextPage && octokitHasNextPage(response)) {
+    response = await octokitGetNextPage(octokit, response)
```

Comment:
Could this pass through the headers to address #760?

Reply:
ahh now I understand the problem with #760. We unfortunately cannot pass headers because we don’t have access to them, the now deprecated `.paginate()` function receives the request promise, not the request options, and we cannot deviate passed headers from the request promise I’m afraid.